docker/docker-compose.dev.yml

```diff
@@ -4,8 +4,14 @@ services:
     oms-events:
         build:
             context: ./${PATH_OMS_EVENTS}/..
-            dockerfile: ./docker/oms-events/Dockerfile.dev
+            dockerfile: ./docker/oms-events/Dockerfile
         image: aegee/oms-events:dev
         volumes:
             - /usr/app/src/node_modules
             - ./${PATH_OMS_EVENTS}/../:/usr/app/src
+
+    oms-events-static:
```

Comment:
imho we don't need one for dev

Reply:
So you suggest to use the default nginx one? The one I've made has some custom logic inside, that's why I'm not using the default dockerfile

Reply:
does it differ between dev and prod?

Reply:
that's more reasonable that what I've suggested then, will update.